dynamodb/src/test/scala/uk/gov/nationalarchives/DADynamoDBClientTest.scala

```diff
@@ -486,6 +486,31 @@ class DADynamoDBClientTest
     queryItemsResponse.head.mockAttribute2 should equal("mockAttributeValue2")
   }
 
+  "queryItems" should "call the query method twice if there are more items to be fetched" in {
+    def items(idx: Int) = Map(
+      "mockAttribute" -> AttributeValue.builder.s(s"mockAttributeValue$idx").build
+    ).asJava
+
+    val itemsRemaining =
+      CompletableFuture.completedFuture(QueryResponse.builder.items(items(0)).lastEvaluatedKey(items(0)).build)
+    val noneRemaining = CompletableFuture.completedFuture(QueryResponse.builder.items(items(1)).build)
```

Comment:
I'll do `noneRemainingResponse` because I would like to reduce the variable lengths a lot so preferably three words or fewer. In context I think it makes sense.